src/index.html

```diff
@@ -7,8 +7,94 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img src="./images/logo.png" alt="logo-img">
+      </a>
+    
+      <nav class="nav">
+        <ul class="list">
+          <li class="value">
+            <a 
+              class="link is-active" 
+              href="#apple"
+            >
+              apple
```

Comment:
```suggestion
              Apple
```
it's better to write first capital